### PR TITLE
changed url

### PR DESCRIPTION
**Pull request by nellarro**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Description

Changed urls on lines 1140 and 1169 from *<a href=\"https://atjonathan.github.io/catphotoapp\* to *<a href=\"http://freecatphotoapp.com/*

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/applied-visual-design.json [hunk 1]
```diff
@@ -1137,7 +1137,7 @@
         "  ",
         "  ",
         "</style>",
-        "<a href=\"https://atjonathan.github.io/catphotoapp\" target=\"_blank\">CatPhotoApp</a>"
+        "<a href=\"http://freecatphotoapp.com/" target=\"_blank\">CatPhotoApp</a>"
```
Comment (JosephLivengood) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: The endquote needs escaped like
` "<a href=\"http://freecatphotoapp.com/\" target=\"_blank\">CatPhotoApp</a>" `

File: seed/challenges/01-responsive-web-design/applied-visual-design.json [hunk 2]
```diff
@@ -1166,7 +1166,7 @@
...
```
Comment (JosephLivengood) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: The endquote needs escaped :+1:

**Review by JosephLivengood — CHANGES_REQUESTED** [review]
Also, please enter the issue number this is fixing please!

File: seed/challenges/01-responsive-web-design/applied-visual-design.json (hunk 2 above)
Comment (nellarro) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: Oh, duh! Lol. Let me change that! Thanks for the heads up!

**Pull request closed without merging**